src/__tests__/App.test.js

```diff
@@ -0,0 +1,49 @@
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { act } from 'react-dom/test-utils';
+import App from '../App';
+import teamsReducer from '../Redux/teams';
+
+const ReactTestRenderer = require('react-test-renderer');
+
+// Mock store
```

Comment:
- [ ] still, one of your tests fails please try to make it pass as the previous reviewer stated in order to cover your app with tests 
<img width="996" alt="Screen Shot 2022-10-14 at 20 03 04" src="https://user-images.githubusercontent.com/61790587/195922334-1aa6ed95-a3f4-48da-8e20-81a7cdf5c05e.png">